src/backend-proxy.js

```diff
@@ -47,10 +48,24 @@ function tryReadData(options, backendResponse, request, next) {
 
 function createHandler({request, response, next, options, backendHttpOptions}) {
 	return backendResponse => {
+		let interceptErrors = false;
+
 		// We always want to copy the status code back to the client
 		response.statusCode = backendResponse.statusCode;
 
 		// Should we intercept the error and raise it as an express error
+
+		if (typeof options.interceptErrors === 'function') {
+			interceptErrors = options.interceptErrors(backendResponse);
+		} else if (typeof options.interceptErrors === 'boolean') {
+			interceptErrors = options.interceptErrors;
+		}
```

Comment:
You can omit this second check in favour of just resolving whether `interceptErrors` is true

```suggestion
		} else {
			interceptErrors = options.interceptErrors === true;
		}
```